plc4j/drivers/opcua/src/main/java/org/apache/plc4x/java/opcua/context/SecureChannel.java

```diff
@@ -361,16 +362,17 @@ public void onConnectOpenSecureChannel(ConversationContext<OpcuaAPU> context, Op
                             LOGGER.error("Failed to connect to opc ua server for the following reason:- {}, {}", ((ResponseHeader) fault.getResponseHeader()).getServiceResult().getStatusCode(), OpcuaStatusCode.enumForValue(((ResponseHeader) fault.getResponseHeader()).getServiceResult().getStatusCode()));
                         } else {
                             LOGGER.debug("Got Secure Response Connection Response");
-                            try {
-                                OpenSecureChannelResponse openSecureChannelResponse = (OpenSecureChannelResponse) message.getBody();
-                                ChannelSecurityToken securityToken = (ChannelSecurityToken) openSecureChannelResponse.getSecurityToken();
-                                tokenId.set((int) securityToken.getTokenId());
-                                channelId.set((int) securityToken.getChannelId());
-                                lifetime = securityToken.getRevisedLifetime();
```

Comment:
It seems like `git merge` has somehow removed this line... I'll add one more commit to fix this.

Reply:
Done in 434acb5